app/src/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum ConnectorsEnum {
```

Comment:
Is this useful? It's not type-safer (the env var is going to be stringly typed anyway) and it's not clearer. I think a plain string comparison should be enough.

Reply:
Done!